depot/containerstore/dependencymanager.go

```diff
@@ -47,14 +48,20 @@ func (bm *dependencyManager) DownloadCachedDependencies(logger lager.Logger, mou
 	completed := 0
 	mountChan := make(chan *cachedBindMount, total)
 	errChan := make(chan error, total)
+	cancelChan := make(chan struct{}, 0)
 	bindMounts := NewBindMounts(total)
 
 	if total == 0 {
 		return bindMounts, nil
 	}
 
+	var wg sync.WaitGroup
+
 	for i := range mounts {
+		wg.Add(1)
```

Comment:
I'm not the most experienced with wait groups, but should this go inside the goroutine? Something like:
```golang
go func(mount *executor.CachedDependency) {
    wg.Add(1)
    defer wg.Done()
   ...
```

Reply:
I think it is the same, since the next line is kicking off the goroutine.